### ISSUE TITLE
Remove deprecated twisted.enterprise.row

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4109]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#4109|
|Type|task|
|Created|2009-11-13 20:55:57Z|

[twisted/enterprise/row.py](twisted/enterprise/row.py) is deprecated, let's remove it.

<details><summary>Searchable metadata</summary>

```
trac-id__4109 4109
type__task task
reporter__thijs thijs
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1258145757000000 1258145757000000
changetime__1325855777000000 1325855777000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_1"></a>|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

Uses util from [#4111](https://github.com/twisted/twisted/issues/4111)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)<a name="note_3"></a>|@itamarst set status to `closed`|
|-|-|

(In [33412]) Merge enterise-util-4111-2.

Author: thijs, itamar
Review: khorn, exarkun, TimAllen
Fixes: [#4108](https://github.com/twisted/twisted/issues/4108), [#4109](https://github.com/twisted/twisted/issues/4109), [#4110](https://github.com/twisted/twisted/issues/4110), [#4111](https://github.com/twisted/twisted/issues/4111)

Remove deprecated code from the enterprise package, along with relevant documentation and examples.
